DAO.sol

```diff
@@ -615,6 +695,47 @@ contract DAO is DAOInterface, Token, TokenCreation {
         p.open = false;
     }
 
+    function withdraw(
+        uint _proposalID,
```

Comment:
@CJentzsch Why do you give `_proposalID` here? It's not used inside the function.